src/Uno.UI.Composition/Composition/CompositionPropertySet.cs

```diff
@@ -80,12 +80,13 @@ private void SetValue<T>(string propertyName, T value)
 		}
 
 		// TODO
```

Comment:
Still `//TODO` ? If not, why not just remove the override?

Reply:
Yup I think we might not need special handling for CompositionPropertySet as the virtual implementation should be doing the right thing. Removed the override. We can adjust if we discovered any needed special handling.